tests/modules/programs/rbw/settings.nix

```
@@ -0,0 +1,34 @@
+{ pkgs, ... }:
+let
+  expected = ''
+    {
+      "base_url": "bitwarden.example.com",
+      "email": "name@example.com",
+      "identity_url": "identity.example.com",
+      "lock_timeout": 300,
+      "pinentry": "@pinentry-gnome@"
+    }
+  '';
+in {
+  config = {
+    programs.rbw = {
+      enable = true;
+      settings = {
+        email = "name@example.com";
+        base_url = "bitwarden.example.com";
+        identity_url = "identity.example.com";
+        lock_timeout = 300;
+        pinentry = "gnome";
+      };
+    };
+
+    nixpkgs.overlays = [ (import ./overlay.nix) ];
+
+    nmt.script = ''
+      assertFileExists home-files/.config/rbw/config.json
+      assertFileContains \
+        home-files/.config/rbw/config.json \
+        '${expected}'
```

```suggestion
      assertFileContent \
        home-files/.config/rbw/config.json \
        '${expected}'
```

Sorry, I meant `assertFileContent` yesterday.